### PR TITLE
Mention use of cast in enum type stub example

### PR DESCRIPTION
**Pull request by hauntsaninja**
In case people gloss over the text, it's good for the example to recommend the thing with the most static information

### CODE REVIEW

File: docs/spec/enums.rst [hunk 1]
```diff
@@ -123,8 +123,9 @@ statically in cases where dynamic values are used.
         genus: str  # Non-member attribute
         species: str  # Non-member attribute
 
-        CAT = ...  # Member attribute
-        DOG = ...  # Member attribute
+        CAT = 1  # Member attribute with known value and type
+        DOG = cast(str, ...)  # Member attribute with unknown value and known type
```
Comment (carljm) [inline docs/spec/enums.rst:127]: As a reader, I think I would be confused by the use of `str` here, given that this appears to be a reimplementation of the above `Pet` enum. Why not just use `int` and keep things consistent?

Comment (hauntsaninja) [inline docs/spec/enums.rst:127]:
```suggestion
        DOG = cast(int, ...)  # Member attribute with unknown value and known type
```

Comment (hauntsaninja) [reply docs/spec/enums.rst]: Changed it! I was anchored to some of the conversations we had around using `_value_` as a way to spell this (which would work for homogenous types, but not heterogeneous types)

**Review by carljm — APPROVED** [review]
Looks fine to me! This is a pretty minor change, but I guess technically any spec change is supposed to go to the whole typing council? Might be good to get at least one more opinion?

**Comment by erictraut** [comment]
Looks good to me too.

**Comment by JelleZijlstra** [comment]
I think this is fine to merge without the full approval procedure because it doesn't change any normative part of the spec, it just clarifies an example.

**Pull request merged**